### PR TITLE
Add where clause integration tests, add support for int64 comparisons

### PR DESCRIPTION
**Pull request by corylanou**

### CODE REVIEW

File: influxql/ast.go [hunk 1]
```diff
@@ -2533,6 +2533,36 @@ func evalBinaryExpr(expr *BinaryExpr, m map[string]interface{}) interface{} {
 			}
 			return lhs / rhs
 		}
+	case int64:
+		// we parse all number literals as float 64, so we have to convert from
+		// an interface to the float64, then cast to an int64 for comparison
+		rhsf, _ := rhs.(float64)
+		rhs := int64(rhsf)
+		switch expr.Op {
+		case EQ:
```
Comment (otoolep) [inline influxql/ast.go]: Yay boilerplate. One thing Go does so well. :-)

**Comment by otoolep** [comment]
+1

File: cmd/influxd/run/server_test.go [hunk 2]
```diff
@@ -1662,3 +1662,199 @@ func TestServer_Query_AcrossShardsAndFields(t *testing.T) {
...
+	test.addQueries([]*Query{
+		// non type specific
+		&Query{
+			name:    "missing measurement with group by",
+			params:  url.Values{"db": []string{"db0"}},
+			command: `SELECT load from missing group by *`,
+			exp:     `{"results":[{"error":"measurement not found: \"db0\".\"rp0\".missing"}]}`,
+		},
+
+		//string
```
Comment (otoolep) [inline cmd/influxd/run/server_test.go]: As trivial as it gets, whitespace before "string".

**Comment by dgnorton** [comment]
+1

**Pull request merged**